tests/alfacase/test_generate_case_description_docstring/test_generate_case_description_docstring_ValveEquipmentDescription_.txt

```diff
@@ -32,5 +32,11 @@
                 value: number
                 unit: string # optional
             opening_curve_interpolation_type: \ :class:`InterpolationType <alfasim_sdk._internal.constants.InterpolationType>`\ # optional
-            opening_curve: \ :class:`opening_curve_description_schema <OpeningCurveDescription>`\ # optional
+            opening_curve: 
+                image:
+                    values: [number]
```

Comment:
Is it simple to change this to `[numbers]` (plural)?

Reply:
Yes, it is easy.
But I read `[number]` as "a list of elements with type number" ("a list of numbers").
The same way I will read `[str]` as "a list of strings".

Reply:
Ahhh I thought it was a placeholder like `<number>`. OK my misunderstanding, please leave as is.